### ISSUE TITLE
Exception on save Configure specific access rights

### ISSUE THREAD

**Issue by bbuerger** [jenkinsci/ownership-plugin#85]
Jenkins (2.281) + Ownership Plugin (0.13) on Windows Server 2016
Role based authentication is used
Click Configure specific access rights -> Use item-specific security -> Save
The attached exception occurs:
[Caught unhandled exception.txt](https://github.com/jenkinsci/ownership-plugin/files/6075442/Caught.unhandled.exception.txt)

**Comment by oleg-nenashev** [comment]
Looks like you submit the project-based security settings before setting up ownership for the item

**Comment by bbuerger** [comment]
You are right: The ownership was definied by the folder (Ownership is defined by: FolderRestricted)
After setting the ownership manually for the item it worked.
Thanks for the quick response (but IMHO setting the ownership via the folder should work as well)

**Comment by bbuerger** [comment]
Saving worked, but when I set the inheritance strategy to:
Do not inherit permission grants from other ACLs
and login as a user, which is not an owner, I am allowed to do anything as before

**Comment by atai** [comment]
Fixed in [jenkinsci/ownership-plugin/pull/101](https://github.com/jenkinsci/ownership-plugin/pull/101)
